lib/draft-live-sync-class.php

```diff
@@ -102,7 +102,6 @@ public function init() {
                 add_action( 'parse_request', array( &$this, 'parse_requests'));
                 add_filter( 'gettext', array( &$this, 'change_publish_button'), 10, 2 );
                 add_filter( 'get_sample_permalink', array( &$this, 'set_correct_permalink'));
-                add_filter( 'page_link', array( &$this, 'set_correct_permalink'));
```

Comment:
The filter that caused permalinks in the sync settings to include full urls with the set preview URL. Removed so that sync works again, but has the drawback that Gutenberg editor does not seem to use it for the Front Page. I guess we have to live it for now.